addons/stock/models/stock_move.py

```diff
@@ -225,6 +228,11 @@ def _compute_reference(self):
         for move in self:
             move.reference = move.picking_id.name if move.picking_id else move.name
 
+    @api.depends('move_line_ids')
+    def _compute_has_move_lines(self):
+        for move in self:
+            move.has_move_lines = len(move.move_line_ids)
```

Comment:
I think `bool` would make more sense (although `len` is ok):
`move.has_move_lines = bool(move.move_line_ids)`